data/url-handlers/anaconda-storage.desktop

```diff
@@ -0,0 +1,6 @@
+[Desktop Entry]
+Name=Anaconda Storage
```

Comment:
Does this appear anywhere visible to the user? I.e. does this need i18n?

Reply:
nope, it's NoDisplay=true, shouldn't show up anywhere in the ui